### ISSUE TITLE
[Feature Request] Node to connect to tfboard, wandb, etc

### ISSUE THREAD

**Issue by clyce** [kijai/ComfyUI-FluxTrainer#79]
Thank you for this excellent extension! In the extension there is a node called visualize training loss, but it only updates on specific triggering(in the example, calling every 750 steps). It would be very nice if the training of the model can be tracked in realtime by tfboard or weight-and-bias.

**Comment by kijai** [comment]
I agree it would, I'm pretty new to training in general and have no experience with those tools, I should really learn. I imagine they would be usable like usual as the nodes just use kohya's training scripts, I may have disabled some wandb calls in effort to get this running in the first place, but most should still be in the code...

**Comment by clyce** [comment]
If the original code in kohya's training scripts contains it, I believe providing some input field on the "FluxTrainLoop" node and pass it to the corresponding function will work

**Comment by plugcrypt** [comment]
I hope to see this feature on the trainer soon, thank you +1
